source/client/output_transform_main.cc

```diff
@@ -0,0 +1,80 @@
+#include "client/output_transform_main.h"
+
+#include "nighthawk/common/exception.h"
+
+#include "external/envoy/source/common/protobuf/message_validator_impl.h"
+
+#include "api/client/service.pb.h"
+
+#include "common/utility.h"
+
+#include "client/factories_impl.h"
+#include "client/options_impl.h"
+
+#include "absl/strings/strip.h"
+#include "tclap/CmdLine.h"
+
+namespace Nighthawk {
+namespace Client {
+
+OutputTransformMain::OutputTransformMain(int argc, const char* const* argv, std::istream& input)
+    : input_(input) {
+  const char* descr = "L7 (HTTP/HTTPS/HTTP2) performance characterization transformation tool.";
+  TCLAP::CmdLine cmd(descr, ' ', "PoC"); // NOLINT
+
+  std::vector<std::string> output_formats = {"human", "yaml", "json"};
+  TCLAP::ValuesConstraint<std::string> output_formats_allowed(output_formats);
+  output_format_ = "";
+  TCLAP::ValueArg<std::string> output_format(
+      "", "output-format",
+      fmt::format("Verbosity of the output. Possible values: [human, yaml, json]."), true, "",
```

Comment:
Is this verbosity or format?

Reply:
Whoops, fixed

Reply:
Also, added a way to pull the supported formats as a string vector in a canonical fashion.